nengo_loihi/loihi_interface.py

```diff
@@ -382,13 +382,13 @@ class LoihiSimulator(object):
 
         .. warning :: Setting the seed has no effect on stochastic
                       operations run on the Loihi board.
-    snip_max_spikes_per_step : int, optional (Default: 50)
-        Maximum number of spikes that can be sent through
-        the nengo_io_h2c channel on one timestep.
     """
 
-    def __init__(self, cx_model,
-                 use_snips=True, seed=None, snip_max_spikes_per_step=50):
+    # Maximum number of spikes that can be sent through
+    # the nengo_io_h2c channel on one timestep.
+    snip_max_spikes_per_step = 50
```

Comment:
This reduces functionality, IMO. We can no longer have multiple simulators with different values for this. Not that that is something that we'd ordinarily like to do, but definitely having it as a class attribute does not fit with my mental model as well.

I think the main reason that I'm hesitant is that I've had bad experiences having `CxSimulator.strict` being a class attribute. It means that in tests, we have to do a funky try/except to make sure that the value changes back after the test. Personally, I'm just not a fan of it at all.

I know we don't want to have a ton of `Simulator` arguments, but IMO there should be some way to get arguments down to the `LoihiSimulator` or `CxSimulator`.